app/views/shared/_new_evidence_checklist.html.haml

```diff
@@ -0,0 +1,11 @@
+- unless @claim.evidence_checklist_ids.empty?
+  .govuk-summary-card
+    .govuk-summary-card__title-wrapper
+      %h2.govuk-summary-card__title= (t('shared.evidence_checklist.caption'))
+    .govuk-summary-card__content
+      %dl.govuk-summary-list
+        = govuk_summary_list do
```

Comment:
`govuk_summary_list` creates a `<dl>` element with the `govuk-summary-list` class so this is creating a `<dl>` nested within a `<dl>`. Line 6 can be removed.